tickets/src/__tests__/middlewares/providerMiddleware.test.ts

```diff
@@ -58,18 +51,6 @@ describe('provider middleware', () => {
       providerMiddleware(config)({ getState, dispatch })(next)(unlockAction)
     })
 
-    it('should set an error and return if there is no matching provider', done => {
```

Comment:
We do not want this behavior anymore: the event should show even if the user does not have a wallet.